gbfs.md

```diff
@@ -326,7 +326,9 @@ Field Name | REQUIRED | Type | Defines
 `datasets[].versions` | Yes | Array&lt;Object&gt; | Contains one object for each of the available versions of a feed. The array MUST be sorted by increasing MAJOR and MINOR version number. 
 `datasets[].versions[].version` | Yes | String | The semantic version of the feed in the form `X.Y`.                               
 `datasets[].versions[].url` | Yes  | URL | URL of the corresponding `gbfs.json` endpoint.
- 
+`datasets[].area` | OPTIONAL | GeoJSON MultiPolygon | A GeoJSON MultiPolygon that describes the operating area. If `area` is supplied, then the record defines the area in which vehicles can be rented and returned.
```

Comment:
the record defines the area in which vehicles can be rented and returned

Is intended that this area be used as an additional geofencing mechanism?  Or is it only descriptive?

Reply:
I don’t see it as an additional geofencing mechanism. It should rather describe the maximum operating area where stations are located and vehicles can be rented or returned. The geofencing mechanism may then provide more fine-grained areas including slow zones, no-parking zones etc.
It is also optional which would make sense for (the unlikely case of) multi-country systems.

Reply:
My main point is that the text as-is written seems slightly ambiguous to me, in that it's not clear if the area is purely descriptive or if it should be used in controlling operations in some way.  For example, if a referenced system had stations or geofenced operating regions outside the bounds of the specified area, would that be considered an error?

Reply:
Suggestion:

> If `area` is supplied, then the record defines the area that covers where vehicles can be rented, returned or used. This is for discovery and not geofencing. Ex. : if vehicles can be rented and renturned only in a city but used anywhere, that area can be the city. On the contrary, if they can only be used inside a city limit but rented in specific parkings, that area could also be the city.

Reply:
Thanks for the suggestion, sounds good to me. Does that address your concerns @bdferris-v2?

Reply:
I'd propose something like:

> If `area` is supplied, then the record describes the general operating area of the system for the purpose of discovery.  Geographic details of the system's operating restrictions must be explicitly specified using station locations and geofencing zones, where appropriate.

Reply:
Thanks, using that language.